src/app/about/about.component.ts

```diff
@@ -0,0 +1,54 @@
+import { ConfirmApplicationDialogComponent } from './../dialogs/confirm-application-dialog/confirm-application-dialog.component';
+import { MemberContractService } from './../services/member-contract.service';
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatTableDataSource, MatSort, MatDialog, MatDialogConfig } from '@angular/material';
+import { Web3Service } from '../services/web3.service';
+
+@Component({
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.css']
+})
+
+export class AboutComponent implements OnInit {
+
+ 
+
+  displayedColumns = ['alias', 'status', 'block'];
```

Comment:
Ist this stuff needed? Looks like a leftover from another component, please remove unnecessary properties or functions.

Reply:
removed